.github/workflows/locale_po_to_json.yaml

```diff
@@ -0,0 +1,68 @@
+name: "Locale PO to JSON"
+
+on:
+  workflow_dispatch:
+permissions:
+  contents: read
+jobs:
+  locale_po_to_json:
+    if: github.repository_owner == 'ManageIQ'
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: manageiq/postgresql:13
+        env:
+          POSTGRESQL_USER: root
+          POSTGRESQL_PASSWORD: smartvm
+          POSTGRESQL_DATABASE: vmdb_i18n
+        options: --health-cmd pg_isready --health-interval 2s --health-timeout 5s --health-retries 5
+        ports:
+        - 5432:5432
+    env:
+      PGHOST: localhost
+      PGPASSWORD: smartvm
+      RAILS_ENV: i18n
+      SKIP_TEST_RESET: true
+      SKIP_DATABASE_SETUP: true
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up system
+      run: bin/before_install
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: "3.1"
+        bundler-cache: true
+      timeout-minutes: 30
+    - name: Prepare dependencies
+      run: bin/setup
+    - name: Install gettext for msgfmt --check support
+      run: sudo apt-get install -y gettext
```

Comment:
This GHA is largely the same as the [core one that generates the po files](https://github.com/ManageIQ/manageiq/blob/master/.github/workflows/locale_update_all.yaml).  The changes are this section where we install gettext for this GHA only whereas we do it in bin/before_install since we run it in tests for core.

Reply:
Yeah I was just thinking maybe we should add this to before install here as well?

Reply:
I like self contained github actions unless it's really complicated or we need this across github actions.  We can certainly move it when that happens otherwise I think it adds needless complexity in the before_install and hides that dependency from the GHA.

Note, I don't expect people to be running this locally, unlike core where we had the locale update all and a test that does `msgfmt --check` both sharing the same common setup.

Reply:
thoughts?